### ISSUE TITLE
What's the purpose of including cps_data/ cps_raw.csv.gz in repo?

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/taxdata#139]
When I recently synchronized my taxdata repo with the central GitHub taxdata repo, I got these results:
```
Checking out files: 100% (16/16), done.
Fast-forward
 cps_data/cps.csv.gz                   |    Bin 22697312 -> 25571361 bytes
 cps_data/cps_raw.csv.gz               |    Bin 68261640 -> 82746004 bytes
 cps_data/finalprep.py                 |     24 +-
 cps_stage3/cps_benefits_extrap.csv.gz |    Bin 0 -> 2963102 bytes
 cps_stage3/extrapolation.py           |    327 +
 cps_stage3/growth_rates.csv           |     14 +
 puf_stage2/puf_weights.csv            | 431052 +++++++++++++++----------------
 puf_stage2/stage2.py                  |      6 +-
 puf_stage3/puf_ratios.csv             |     29 +-
 puf_stage3/stage3.py                  |      6 +-
 stage1/CBO_baseline.csv               |     32 +-
 stage1/Stage_II_targets.csv           |     62 +-
 stage1/Stage_I_factors.csv            |     27 +-
 stage1/Stage_I_factors_transpose.csv  |     44 +-
 stage1/growfactors.csv                |     27 +-
 stage1/stage1.py                      |      7 +-
 16 files changed, 216003 insertions(+), 215654 deletions(-)
 create mode 100644 cps_stage3/cps_benefits_extrap.csv.gz
 create mode 100644 cps_stage3/extrapolation.py
 create mode 100644 cps_stage3/growth_rates.csv
Counting objects: 250, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (144/144), done.
Writing objects: 100% (250/250), 313.26 MiB | 2.12 MiB/s, done.
Total 250 (delta 151), reused 198 (delta 106)
remote: Resolving deltas: 100% (151/151), completed with 18 local objects.
remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
remote: warning: See http://git.io/iEPt8g for more information.
remote: warning: File cps_data/cps_raw.csv.gz is 78.91 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
To https://github.com/martinholmer/taxdata.git
   190c6f4..dee9dde  master -> master
```
So, is `cps_data/cps_raw.csc.gz` the file from John O'Hare?
Or is it some intermediate results in the data preparation process?
At nearly 79 MB, it is quite a bit bigger than what GitHub recommends as a maximum file size.

**Comment by andersonfrailey** [comment]
Sorry I missed this question, @martinholmer. `cps_raw.csv.gz` was the file we got from John. It's the output of all the SAS scripts. We originally included it because we got the file before we got the SAS scripts and wanted to at least include the file the final prep scripts processed available.

I believe we should leave it for the time being because SAS requires a license to use, so only those who have one would be able to recreate the raw CPS file. Leaving it in the repo allows users to modify the final prep scripts and ultimately the final CPS files to fit their needs.

I do recognize that it is a very large file though. Once we are able to fully recreate the CPS file and CPS weights file using Python and other open source languages (I'm exploring using Julia  to create the weights file), I would have no issue with removing it.
